sdk/maps/maps-render-rest/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 ### Breaking Changes
 
+- Deprecate tilesetId "microsoft.dem" and "microsoft.dem.contours" per [announcement of the retirement for DEM tiles in Render API](https://azure.microsoft.com/updates/azure-maps-elevation-apis-and-render-v2-dem-tiles-will-be-retired-on-5-may-2023/)
```

Comment:
Move it to `Other Changes`. See SDK board's comment in C# SDK: https://github.com/Azure/azure-sdk-for-net/pull/37112#discussion_r1237110972